pkg/command/cdc/create.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2021 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cdc
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"knative.dev/kn-plugin-admin/pkg"
+	typev1alpha1 "knative.dev/networking/pkg/apis/networking/v1alpha1"
+)
+
+// NewCdcCreateCommand to create cluster domain claims
+func NewCdcCreateCommand(p *pkg.AdminParams) *cobra.Command {
+	var namespace string
+	cdcCreateCommand := &cobra.Command{
+		Use:   "create",
+		Short: "create cluster domain claim",
+		Long:  "Create Knative cluster domain claim",
+		Example: `
+  # To create a cluster domain claim
+  kn admin cdc create domain.name --namespace ns1`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			client, err := p.NewNetworkingClient()
+			if err != nil {
+				return err
+			}
+
+			name := args[0]
```

Comment:
I think there's should be a check for exactly one argument provided in the array, otherwise error if empty etc. E.g. similarly to `kn service create` cmd.

That's valid for `delete` cmd as well.

Reply:
Would you mind changing it to `kn` style of error? Sorry I should have add that link to the first comment. :)

https://github.com/knative/client/blob/main/pkg/kn/commands/service/create.go#L91-L93

Reply:
thanks, that looks better. Changed